### ISSUE TITLE
Example of private.js

### ISSUE THREAD

**Issue by heff** [bfricka/less-preview#16]
When I first attempted to run this locally I had to reverse engineer private.js, even though it's not needed locally. Providing and example file of how it should look could be one way to help this. e.g. private.js.example

This is what I put in mine to get it to work.

``` js
module.exports = {
  app: {
    secretKey: 'asdfasdfadsf'
  },
  github: {
    clientID: 'heff',
    callbackURL: 'asdf',
    clientSecret: 'asdf'
  }
};
```

**Comment by bfricka** [comment]
Yeah, that just a piece that I haven't even implemented.  The page gets lots of hits, but I'm pretty sure you're the first to actually use the code for anything, so I have crap in here that really shouldn't be in a public release.  I started a github auth so people can sign in and access their less files, but I ended up losing focus and merging that branch for its useful changes w/out cleaning out the incomplete ones.

I can strip this out at some point.
